Header header
geometry_msgs/Transform   T_w_l
uint32    occupied_cell_count
uint32[]  occupied_cell_idx
